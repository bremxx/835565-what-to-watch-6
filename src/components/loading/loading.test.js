import React from 'react';
import {render} from '@testing-library/react';
import {Router} from 'react-router-dom';
import Loading from './loading';
import {createMemoryHistory} from 'history';

test(`Loading component render test by snapshot`, () => {
  const history = createMemoryHistory();
  const {container} = render(
      <Router history={history}>
        <Loading />
      </Router>
  );
  expect(container).toMatchSnapshot();
});
