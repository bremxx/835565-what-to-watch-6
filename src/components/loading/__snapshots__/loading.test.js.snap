// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component render test by snapshot 1`] = `
<div>
  <div
    style="width: 100%; height: 25vh; position: relative;"
  >
    <div
      class="lds-dual-ring"
    />
  </div>
</div>
`;
